src/pudl/transform/eia861.py

```diff
@@ -1233,13 +1202,6 @@ def demand_response(tfr_dfs):
     * Tidy subset of the data by customer class.
     * Drop duplicate rows based on primary keys.
     * Convert 1000s of dollars into dollars.
```

Comment:
Can you include a description of the returned dataframes?

Reply:
What kind of descriptive information are you looking for here and below?